opencti-platform/opencti-graphql/src/utils/authorizedMembers.ts

```diff
@@ -10,7 +11,7 @@ export const getAuthorizedMembers = async (
   entity: BasicStoreEntity & { authorized_members: Array<AuthorizedMember> }
 ): Promise<MemberAccess[]> => {
   let authorizedMembers: MemberAccess[] = [];
-  if (!entity.authorized_members?.length) {
+  if (isEmptyField(entity.authorized_members)) {
```

Comment:
`isEmptyField` uses Ramda.

Reply:
Yes, but isEmptyField is our prefered method at the moment for checking that.